hooks/airbrake/airbrake.go

```diff
@@ -1,51 +1,51 @@
-package logrus_airbrake
+package airbrake
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/tobi/airbrake-go"
```

Comment:
I considered that, but found it wasn't a problem. When referring to `tobi`'s package, it's clear that we're referring to an external package.

Also, `tobi`'s package is no longer imported when using the Airbrake hook since the Airbrake credentials are now scoped to the `AirbrakeHook` struct (which could also have been confusing).